bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -0,0 +1,59 @@
+"""
+Use ``nose`` to run these unit tests.
+"""
+import uuid
+
+from . import GalaxyTestBase, test_util
+
+
+@test_util.skip_unless_galaxy()
+class TestGalaxyQuotas(GalaxyTestBase.GalaxyTestBase):
+
+    def setUp(self):
+        super(TestGalaxyQuotas, self).setUp()
+        # Quota names must be unique, and they're impossible to delete
+        # without accessing the database.
+        self.quota_name = 'BioBlend-Test-Quota-%s' % uuid.uuid4().hex
+        self.quota = self.gi.quotas.create_quota(
+            self.quota_name, 'testing', '100 GB', '=',
+            default='registered'
+        )
+
+    def tearDown(self):
+        self.gi.quotas.update_quota(self.quota['id'], default='registered')
```

Comment:
Uh, awful! What about changing the method docstring from:

> Calling update twice with a default of ``no`` will throw an error.

to: 

> Calling this method with ``default="no"`` on a non-default quota will throw an error.

? I think this wording includes also the case where the quota is directly created with ``default="no"`` and then updated.